package.json

```diff
@@ -17,6 +17,7 @@
     "lint-fix-css": "yarn lint-css --fix",
     "prettier-json": "prettier --write src/**/*.json",
     "flow": "flow --max-warnings 0",
+    "flow:ci": "flow check --max-warnings 0",
```

Comment:
I realized from a comment in an issue on flow repository that `flow check` doesn't start up a server. This makes it unnecessary to run `flow-stop` in CI.
Moreover when starting a server we never could execute the next line in appveyor. I don't know if this means that there's a bug in Flow on windows, but at least this fixes this for us.

Reply:
I really wish we could document and add comments to our yarn run commands.

Reply:
Maybe with keys like `"flow:ci:about"` ?